draft-west-cookie-incrementalism.md

```diff
@@ -188,9 +188,41 @@ to:
 
 ~~~
 13. If the cookie-attribute-list contains an attribute
-    with an attribute-name of "SameSite", set the
+    with an attribute-name of "SameSite" and an
+    attribute-value of "Strict", "Lax", or "None", set the
     cookie's same-site-flag to attribute-value. Otherwise,
-    set the cookie's same-site-flag to "Lax".
+    set the cookie's same-site-flag to "Unset".
```

Comment:
It's a little strange here that we end up with "Default" above and "Unset" here. It makes sense, given the flag names, but perhaps we should use "Default" here as well for clarity?

Reply:
Changed to "Default".